.github/workflows/conda-env-create.yml

```diff
@@ -4,27 +4,42 @@ name: Solve Conda Environment
 
 on:
   push:
-    paths: [ "requirements*.yml", "conda-env-create.yml" ]
+    paths: [ "requirements*.yml", "conda-env-create.yml", "requirement*.txt"]
   pull_request:
-    paths: [ "requirements*.yml", "conda-env-create.yml" ]
+    paths: [ "requirements*.yml", "conda-env-create.yml", "requirement*.txt"]
   schedule: # Run on the 1st of every month at midnight
     - cron: "0 0 1 * *"
 
 jobs:
   conda_solve:
+    defaults:
+      run:
+        working-directory: .
     strategy:
       fail-fast: true
       matrix:
         os: [ubuntu-latest, windows-latest]
+        kind: ["dev", "prod"]
         include:
           - os: ubuntu-latest
+            kind: prod
             requirements: requirements.conda.yml
+          - os: ubuntu-latest
+            kind: dev
+            requirements: requirements.dev.conda.yml
           - os: windows-latest
+            kind: prod
             requirements: requirements.win64.conda.yml
+          - os: windows-latest
+            kind: dev
+            requirements: requirements.win64.dev.conda.yml
     runs-on: ${{ matrix.os }}
     timeout-minutes: 20
     steps:
       - uses: actions/checkout@v2
+      - name: Copy file
```

Comment:
```suggestion
      - name: Copy requirements files
```

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/455/commits/59f01e6e48eadfc58a736558303f09cbc16276b0